subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/AccessorsClassPath.kt

```diff
@@ -38,41 +32,28 @@ import org.gradle.internal.execution.WorkExecutor
 import org.gradle.internal.execution.history.ExecutionHistoryStore
 import org.gradle.internal.execution.history.changes.InputChangesInternal
 import org.gradle.internal.file.TreeType
-import org.gradle.internal.fingerprint.CurrentFileCollectionFingerprint
-import org.gradle.internal.fingerprint.FileCollectionFingerprint
-import org.gradle.internal.fingerprint.FileCollectionSnapshotter
-import org.gradle.internal.fingerprint.impl.OutputFileCollectionFingerprinter
```

Comment:
Can we remove now OutputFileCollectionFingerprinter and `OutputNormalizer`?

Reply:
We still use them because the output properties are defined with `OutputNormalizer` as their normalization. I chose not to remove that given that we likely going to remove fingerprinting for outputs in one way or another soon. We can handle this then.